ui/src/dashboards/components/Visualization.js

```diff
@@ -63,6 +65,7 @@ DashVisualization.propTypes = {
       bounds: arrayOf(string),
     }),
   }),
+  tableOptions: shape({}),
```

Comment:
at some point we will spell this out, but.... we don't have all of the options and their types yet :)